components/applications-service/integration_test/global_test.go

```diff
@@ -413,3 +423,109 @@ func habServicesABCD() []*habitat.HealthCheckEvent {
 		),
 	}
 }
+
+// @afiune there are cases where the order of the services that are returned are not
+// exactly the same, we do care about order but in some degree, for example health check
+// status and fqdn, but we don't care about things like order of supervisor_id so we can
+// skip that check by not specifying it into the expected response
+func assertServicesEqual(t *testing.T, expected, actual []*applications.Service) {
```

Comment:
❓ `assert.ElementsMatch(t, expected, actual)` doesn't do the trick?

Reply:
First of all, thanks for looking into this PR! Second, this was just a move of code from A to B file https://github.com/chef/automate/pull/839/commits/06b87a19ab2b7aba63660214cc2da1100a6844bd
Third and last, no, it doesn't do the trick since we might have db IDs and timestamps that won't match exactly the same.

Reply:
Ah, I see. Thanks.